docs/en/setup/service-agent/java-agent/Plugin-list.md

```diff
@@ -115,4 +115,5 @@
 - zookeeper-3.4.x
 - mssql-jtds-1.x
 - mssql-jdbc
-- apache-cxf-3.x
\ No newline at end of file
+- apache-cxf-3.x
+- dolphinscheduler
```

Comment:
Don't add this. This is the CI verification file, which relates to the main repo plugin.